packages/firebase_ml/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -38,10 +40,12 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
-        <!-- Don't delete the meta-data below.
```

Comment:
Is there a reason you're using the v1 embedding instead of v2?

Reply:
My understanding is that this way I  test v1 embedding’s compatibility with my plugin. Same thing is done in all other packages in the repo.

Reply:
I think you have to include both Activities in the `AndroidManifest` like in the `firebase_ml_vision` plugin: https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_ml_vision/example/android/app/src/main/AndroidManifest.xml#L20